package.json

```diff
@@ -27,31 +28,30 @@
     "url": "https://github.com/iTowns/itowns2/issues"
   },
   "homepage": "https://itowns.github.io/",
+  "dependencies": {
```

Comment:
There is a return on the `dependencies`? 
[https://github.com/iTowns/itowns2/pull/241/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L23](https://github.com/iTowns/itowns2/pull/241/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L23)